src/components/BookForm/BookForm.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Row from "react-bootstrap/Row";
+import "./BookForm.css";
+
+function BookForm() {
+  return (
+    <Form>
+      <h2 className="head">Book Form</h2>
+      <Row className="m-2">
+        <Form.Group as={Col} controlId="formGridBookName">
+          <Form.Label>Название книги</Form.Label>
+          <Form.Control type="bookname" placeholder="Кобзарь" />
+        </Form.Group>
+
+        <Form.Group as={Col} controlId="formGridAuthorName">
+          <Form.Label>Автор</Form.Label>
+          <Form.Control type="AuthorName" placeholder="Тарас Шевченко" />
+        </Form.Group>
+      </Row>
+
+      <Form.Group className="m-3" controlId="formGrid">
+        <Form.Label>Описание</Form.Label>
+        <Form.Control as="textarea" rows={2} />
+      </Form.Group>
+
+      {['checkbox'].map((type) => (
```

Comment:
Тут нужен map чтобы каждый элемент массива был чекбоксом
Например:
```suggestion
      {['Фантастика', 'Детектив', 'Драма'].map((label) => (
      <Form.Check
            inline
            label={label}
            type='checkbox'
            id={`inline-${type}-3`}
          />
      
      ))
```